docs/cookbook/database-orm.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Adding a Database ORM
+---
+
+You can use a database ORM such as [Eloquent](https://laravel.com/docs/5.1/eloquent) to connect your SlimPHP application to a database.
+
+## Adding Eloquent to your application
+
+<figure>
+{% highlight bash %}
+composer require illuminate/database "~5.1"
+{% endhighlight %}
+<figcaption>Figure 1: Add Eloquent to your application.</figcaption>
+</figure>
+
+## Configure Eloquent
+
+In your `dependencies.php` or wherever you add your Service Factories:
+
+<figure>
+{% highlight php %}
+// -----------------------------------------------------------------------------
+// Service factories
+// -----------------------------------------------------------------------------
+
+$container['db'] = function ($container) {
+    $capsule = new \Illuminate\Database\Capsule\Manager;
+    $capsule->addConnection([
```

Comment:
might be useful to show how you can use Slim's settings array to contain this.

```php
$container['settings']['db'] = [
        'driver' => 'mysql',
         'host' => 'localhost'
         'database' => 'database',
         'username' => 'user',
         'password' => 'password',
         'charset'   => 'utf8',
         'collation' => 'utf8_unicode_ci',
         'prefix'    => '',
];
```
Then it's simple to do, `$capsule->addConnection($container['settings']['db']);`